### ISSUE TITLE
Hex color codes are minimised to 3-character if possible

### ISSUE THREAD

**Issue by adrianforsius** [peterbe/premailer#143]
6 character color codes are reduced to 3 character color codes if possible e.g:
#FFFFFF => #FFFF
#112233 => #123

This is caused by cssutils parser having this as default behaviour, however this is a common mail-client issue:
![screen shot 2015-10-15 at 09 03 14](https://cloud.githubusercontent.com/assets/616959/10506780/bb16782c-731b-11e5-9ad3-8ceb3a070600.png)

cssutils parser function (cssutils/serialize.py):
```python
def _hash(self, val, type_=None):
        """
        Short form of hash, e.g. #123 instead of #112233
        """
        # TODO: add pref for this!
        if len(val) == 7 and val[1] == val[2] and\
                             val[3] == val[4] and\
                             val[5] == val[6]:
            return u'#%s%s%s' % (val[1], val[3], val[5])
        else:
            return val
```

**Comment by elidickinson** [comment]
Do you have an example of where this causes a color to be misinterpreted? I use premailer and gmail a lot and have never noticed any issues. I think perhaps it was a problem a long time ago, but gmail has long since fixed it. (There is a problem with 3 character background colors and Lotus Notes but we worked around it in #114)

**Comment by peterbe** [comment]
I'd love for there to be a thing in premailer that, by default, tells cssutils to *not* bother.

**Comment by peterbe** [comment]
...because the byte-saving isn't worth it.

**Comment by adrianforsius** [comment]
I made a pull-request to cssutils to have preference for that, in the meanwhile I'll use the fix  discussed in #114 . Thanks for feedback.

**Comment by peterbe** [comment]
@adrianforsius cssutils is no longer maintained. The original author gave me write access and admin access to his PyPI page but I only needed it so I could push a critical fix that makes cssutils work in python 3.5.

**Comment by peterbe** [comment]
Should be solved with https://github.com/peterbe/premailer/commit/e5f73b074a4efb7898204e2fb122b62542494a2e
